assets/src/js/batch.jsx

```diff
@@ -140,6 +142,11 @@ var App = React.createClass( {
 		this.toggleProcessing( true );
 	},
 
+	toggleResetModal : function( active ) {
+		if( active === false || active === true ) {
```

Comment:
We do have eslint configured. It looks like it's not failing the `gulp` task, and so it doesn't fail the build. cc @JasonHoffmann